lib/driver.js

```diff
@@ -925,6 +927,17 @@ class XCUITestDriver extends BaseDriver {
       }
     }
 
+    if (!_.isEmpty(caps.browserName)) {
+      if (!_.isEmpty(caps.bundleId)) {
+        log.errorAndThrow(`'browserName' cannot be set together with 'bundleId' capability`);
+      }
+      // warn if the capabilities have both `app` and `browser, although this
+      // is common with selenium grid
+      if (!_.isEmpty(caps.app)) {
+        log.warn('The capabilities should generally not include both an app and a browserName');
```

Comment:
In the previous error `browserName` and `bundleId` are in single quotes. Perhaps they ought to be here, too.

Reply:
changed